applications/rag/tests/test_rag.py

```diff
@@ -62,4 +62,93 @@ def test_prompts(prompt_url):
         for substring in expected_substrings:
             assert substring in text, f"substring {substring} not in response:\n {text}"
 
-test_prompts(sys.argv[1])
+def test_prompts_nlp(prompt_url):
```

Comment:
Expected to fail until we have an image with this change https://github.com/GoogleCloudPlatform/ai-on-gke/pull/513

Reply:
image is updated in main now